doc/source/examples/quickstart.ipynb

```diff
@@ -1,28 +1,29 @@
 {
```

Comment:
<p>Do we want to filter warnings for this?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MDAnalysis/UserGuide/pull/241/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
<p>Fixed.</p>